spec/beef/extensions/websocket_hooked_browser_spec.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+require 'rest-client'
+require 'core/main/network_stack/websocket/websocket'
+require 'websocket-client-simple'
+
+RSpec.describe 'BeEF Extension WebSockets' do
+
+  before(:all) do
+    @config = BeEF::Core::Configuration.instance
+    @cert_key = @config.get('beef.http.https.key')
+    @cert = @config.get('beef.http.https.cert')
+    @port = @config.get('beef.http.websocket.port')
+    @secure_port = @config.get('beef.http.websocket.secure_port')
+    @config.set('beef.http.websocket.secure', true)
+    @config.set('beef.http.websocket.enable', true)
+  end
+
+  it 'can hook a browser with websockets' do
+    # start beef server
+    @config = BeEF::Core::Configuration.instance
```

Comment:
This is un-needed as you've already set @config to the same thing on line 9.

Reply:
Thanks!